tasks/office.py

```diff
@@ -232,5 +286,81 @@ def get_moves_queue(self, overrides=None):
             **self.user.cert_kwargs,
         )
         moves, success = check_response(resp, "getMovesQueue")
+        if success:
+            self.add_stored(MOVE_TASK_ORDER, moves["queueMoves"])
 
-        self.add_stored(MOVE_TASK_ORDER, moves["queueMoves"])
+    @tag(ORDER, "getOrder")
+    @task
+    def get_orders(self, overrides=None):
```

Comment:
+1 that we should think about that. It's tricky though, might need a good brainstorming sesh. We need to remember that we don't have a lot of folks on MM who are super familiar with Python, so we do want to avoid any solutions that are too complex or abstract

Reply:
+1 on the idea of doing some brainstorming on that. I suspect there's a happy place in the abstraction spectrum we can shoot for.

Reply:
Agreed, I'm thinking we may still need to be smarter about how we store things by status and that is a looming thing that may influence how we do things.